packages/cdktf/lib/terraform-stack.ts

```diff
@@ -136,12 +141,34 @@ export class TerraformStack extends Construct {
       : "";
   }
 
+  private findAll<T>(ClassConstructor: Constructor<T>): T[] {
+    const items: T[] = [];
+
+    const visit = async (node: IConstruct) => {
```

Comment:
Does this need to be async? I maybe thinking too much about c#, but seems like this is synchronous.